modules/navigation/register_types.cpp

```diff
@@ -65,15 +62,10 @@ void register_gdnavigation_types() {
 
 #ifdef TOOLS_ENABLED
 	EditorPlugins::add_by_type<NavigationMeshEditorPlugin>();
-
-	ClassDB::APIType prev_api = ClassDB::get_current_api();
-	ClassDB::set_current_api(ClassDB::API_EDITOR);
-
-	ClassDB::set_current_api(prev_api);
```

Comment:
Removed this as this wasn't used. I'm not sure if editor plugins are meant to be registered in the core or editor API, here and in gridmap and csg it's core, in gltf it's editor.